src/main/scala/com/cognite/sdk/scala/common/ReadWritableResource.scala

```diff
@@ -93,20 +91,23 @@ object DeleteByExternalIds {
     // TODO: group deletes by max deletion request size
     //       or assert that length of `ids` is less than max deletion request size
     requestSession.post[Unit, Unit, Items[CogniteExternalId]](
-      Items(externalIds.map(CogniteExternalId)),
+      Items(externalIds.map(CogniteExternalId.apply)),
       uri"$baseUrl/delete",
       _ => ()
     )
 }
 
-trait Create[R, W, F[_]] extends WithRequestSession[F] with CreateOne[R, W, F] with BaseUrl {
+trait Create[R <: ToCreate[W], W, F[_]]
```

Comment:
They do! Using short names is just a convention, https://github.com/scala/docs.scala-lang/blob/main/_style/naming-conventions.md#type-parameters-generics
at least in part because it gets confusing to see `Map[Key, Value]` if you also have an actual class/trait/whatever named `Value`, which will be shadowed by the type parameter `Value` - so `Value` and `V` is actually more clear.

Arguably we could have used `ReadItem`, `WriteItem` here. I think we did at some point, but it was confusing in its own way. Might have been before the restructuring into traits like this.